ImperatorToCK3/Source/CK3/CK3World.h

```diff
@@ -16,7 +16,7 @@ class World
 		World(const ImperatorWorld::World& impWorld, const Configuration& theConfiguration, const mappers::VersionParser& versionParser);
 		World(const ImperatorWorld::World& impWorld): outputModName(impWorld.getSaveName()) {};
 
-		std::string getOutputModName() const { return outputModName; }
+		[[nodiscard]] std::string getOutputModName() const { return outputModName; }
```

Comment:
When returning classes, unlike primitives (bool, int, double), if you're not doing fancy stuff with discardable classes, it's always a good idea to return a reference so no copying is done. And since noone will ever change outputname, that reference can be a const, which helps the compiler a lot. 

This particular instance won't be called more than once so it doesn't matter much, but elsewhere you can get serious savings. So, return const auto& instead of std::string and make a habit of it, whenever nothing else is needed.